packages/mdc-switch/README.md

```diff
@@ -70,17 +74,66 @@ Users can add the `disabled` attribute directly to the `<input>` element or a pa
-`mdc-switch-knob-color($color)` | Sets the knob color.
-`mdc-switch-focus-indicator-color($color)` | Sets the focus indicator color.
+`mdc-switch-toggled-on-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled on.
+`mdc-switch-toggled-off-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled off.
+`mdc-switch-toggled-on-track-color($color)` | Sets color of the track when the switch is toggled on.
+`mdc-switch-toggled-off-track-color($color)` | Sets color of the track when the switch is toggled off.
+`mdc-switch-toggled-on-thumb-color($color)` | Sets color of the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-thumb-color($color)` | Sets color of the thumb when the switch is toggled off.
+`mdc-switch-toggled-on-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled off.
+
+## `MDCSwitch` Properties and Methods
+
+Property | Value Type | Description
+--- | --- | ---
+`checked` | Boolean | Setter/getter for the switch's checked state
+`disabled` | Boolean | Setter/getter for the switch's disabled state
+
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a switch for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
+
+### `MDCSwitchAdapter`
+
+| Method Signature | Description |
+| --- | --- |
+| `addClass(className: string) => void` | Adds a class to the root element. |
+| `removeClass(className: string) => void` | Removes a class from the root element. |
+| `setNativeControlChecked(checked: boolean)` | Sets the checked state of the native control. |
+| `isNativeControlChecked() => boolean` | Returns the checked state of the native control. | 
+| `setNativeControlDisabled(disabled: boolean)` | Sets the disabled state of the native control. |
+| `isNativeControlDisabled() => boolean` | Returns the disabled state of the native control. | 
+
+### `MDCSwitchFoundation`
+
+| Method Signature | Description |
+| --- | --- |
+| `isChecked() => boolean` | Returns whether the native control is checked. |
+| `setChecked(checked: boolean) => void` | Sets the checked value of the native control and updates styling to reflect the checked state. |
+| `isDisabled() => boolean` | Returns whether the native control is disabled. |
+| `setDisabled(disabled: boolean) => void` | Sets the disabled value of the native control and updates styling to reflect the disabled state. |
+| `handleChange() => void` | Handles a change event from the native control. |
+
+### `MDCSwitchFoundation` Event Handlers
```

Comment:
Shouldn't this be documented under the component rather than the foundation?

Reply:
I just re-read this and realized why it's here; I had misread it earlier. This is basically an example of the sort of thing Pat was saying we should add, so this can stay here.